src/Groups/group_characters.jl

```diff
@@ -153,7 +153,7 @@ function character_table(id::String, p::Int = 0)
       modid = id
     else
       isprime(p) || error("p must be 0 or a prime integer")
-      modid = id * "mod" * string(p)
+      modid = "$id mod $p"
```

Comment:
This adds some extra spaces to a string that is used for lookup in a character table. This might cause problems?
(e.g. the docstring above contains `character_table("A5mod2")` which only matches the old output)

```suggestion
      modid = "$(id)mod$(p)"
```

Reply:
Thanks, that was an accident, fixed now